sources/packages/backend/apps/api/src/route-controllers/supporting-user/models/supporting-user.dto.ts

```diff
@@ -66,6 +66,7 @@ export class UpdateSupportingUserAPIInDTO {
 export class ApplicationAPIOutDTO {
   programYearStartDate: string;
   formName: string;
+  offeringIntensity: string
```

Comment:
can we please have the type as OfferingIntensity from @sims/sims-db

Reply:
Thanks for spotting this. I used OfferingIntensity for the frontend and forgot to use it for the backend. It is now using OfferingIntensity from @sims/sims-db.